db-setup.yml

```diff
@@ -1,19 +1,12 @@
+dist: bionic
```

Comment:
I don't know how a potential conflict will be resolved here. Is it the build env that appears last in the file, i.e. dependent on where the shared config is injected? Depending on how this works, it may break other apps that rely on a different version of the build environment. This may not be a big deal to fix though.

@DamianSzymanski Have you tried these changes in isolation on any of the apps that inject this config to make sure it works?